README.md

```diff
@@ -252,10 +261,10 @@ THEN:
 	response body should not contain (.*)
 	response body path (.*) should be (.*)
 	response body path (.*) should not be (.*)
-    	response body path (.*) should be of type array
-    	response body path (.*) should be of type array with length (\d+)
-    	response body should be valid according to schema file (.*)
-    	response body should be valid according to swagger definition (.*) in file (.*)
+   	response body path (.*) should be of type array
+   	response body path (.*) should be of type array with length (\d+)
+   	response body should be valid according to schema file (.*)
+   	response body should be valid according to openapi description (.*) in file (.*)
```

Comment:
I think description is the correct term. Specification is the spec for the description! :)

```
The OpenAPI Specification is a community driven, open specification within the Open API Initiative, a Linux Foundation Collaborative Project.

The OpenAPI Specification (OAS) defines a standard, programming language-agnostic interface description for REST APIs, which allows both humans and computers to discover and understand the capabilities of a service without requiring access to source code
```
https://github.com/OAI/OpenAPI-Specification